spec/index.html

```diff
@@ -725,6 +733,8 @@ <h2>Selected Terminal Literal Strings</h2>
       <dd>three concatenated quotation mark characters, each having the code point <code class="codepoint">U+0022</code></dd>
       <dt id="cp-triple-apostrophe"><code>'''</code></dt>
       <dd>three concatenated apostrophes characters, each having the code point <code class="codepoint">U+0027</code></dd>
```

Comment:
Either `apostrophe characters` or `apostrophes`.
```suggestion
      <dd>three concatenated apostrophe characters, each having the code point <code class="codepoint">U+0027</code></dd>
```